### PR TITLE
Drv scpi

### PR DESCRIPTION
**Pull request by luroche**
Working with example of flow.

### CODE REVIEW

File: code/drv_scpi/src/scpi_sniffer/drv_scpi_cmd.py [hunk 1]
```diff
@@ -54,16 +54,24 @@ def __init__(self, data_type: DrvScpiCmdTypeE, port: str, **kwargs:any):
...
-                not isinstance(self.payload, DrvScpiSerialConfC):
-                log.error("Payload must be a DrvScpiSerialConfC object")
-                raise TypeError("Payload must be a DrvScpiSerialConfC object")
-            elif (self.data_type in (DrvScpiCmdTypeE.WRITE, DrvScpiCmdTypeE.WRITE_READ, \
-                DrvScpiCmdTypeE.RESP)) and not isinstance(self.payload, str):
+            if self.data_type == DrvScpiCmdTypeE.ADD_DEV:
+                if not isinstance(self.payload, DrvScpiSerialConfC):
+                    log.error("Payload must be a DrvScpiSerialConfC object")
+                    raise TypeError("Payload must be a DrvScpiSerialConfC object")
+                elif not hasattr(self, 'rx_chan_name'):
```
Comment (mariuscrsn) [inline code/drv_scpi/src/scpi_sniffer/drv_scpi_cmd.py:61]: Check type of rx_chan_name is str

File: code/drv_scpi/src/scpi_sniffer/drv_scpi_iface.py [hunk 2]
```diff
@@ -118,23 +119,36 @@ def read(self) -> None:
         Raises:
             - None.
         '''
-        log.info(f"Reading port: {self.__rx_chan_name}...") # pylint: disable=logging-fstring-interpolation
         msg_read = self.__serial.readline() #TODO: Check if this is the best way to read
+        # msg_read = [msg_read.decode("utf-8")]
+        log.warning(f"Port: {self.__rx_chan_name} \t Message read RX: {msg_read}")
+        # DrvScpiCmdDataC(port = self.__serial.port, data_type = DrvScpiCmdTypeE.RESP,
```
Comment (mariuscrsn) [inline code/drv_scpi/src/scpi_sniffer/drv_scpi_iface.py]: Remove comments

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/drv_scpi/src/scpi_sniffer/drv_scpi_cmd.py (hunk 1 above)
Comment (luroche) [reply code/drv_scpi/src/scpi_sniffer/drv_scpi_cmd.py:61]: Done.

File: code/drv_scpi/src/scpi_sniffer/drv_scpi_iface.py (hunk 2 above)
Comment (luroche) [reply code/drv_scpi/src/scpi_sniffer/drv_scpi_iface.py]: Done.

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**